### PR TITLE
Added pylinter.sh and setup.cfg

### PR DESCRIPTION
**Pull request by Man-Jain**
Added a bash script for checking for linting errors.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[pycodestyle]
+ignore = W504
+max-line-length = 80
+
+
```
Comment (sara-02) [inline setup.cfg]: please remove this extra new line from here.

File: pylinter.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+fail=0
+
+function prepare_venv() {
+    VIRTUALENV=$(which venv)
+    if [ $? -eq 1 ]; then
+        
+        VIRTUALENV=$(which venv-3)
+    fi
+
+    ${VIRTUALENV} python3 -m venv mybenjienv && source myenv/bin/activate && python3 "$(which pip3)" install pycodestyle
```
Comment (sara-02) [inline pylinter.sh]: Since you are naming your venv `mybenjienv`, `source`  should also use the same name and not `myenv`

**Comment by Man-Jain** [comment]
Sorry for those silly mistakes will fix them now.

**Comment by sara-02** [comment]
@the-ethan-hunt We are good to merge this now.
@Man-Jain You will have to send another set of PR with the code fixes as well. Please add that PR as soon as possible.

**Comment by sara-02** [comment]
Fixes issue #132

**Pull request merged**